### ISSUE TITLE
manifest.json needs a version key

### ISSUE THREAD

**Issue by juan11perez** [bimmerconnected/ha_custom_component#65]
Please see below error on start up.

```
2023-06-03 12:43:22.553 ERROR (SyncWorker_0) [homeassistant.loader] The custom integration 'bmw_connected_drive' does not have a version key in the manifest file and was blocked from loading. See https://developers.home-assistant.io/blog/2021/01/29/custom-integration-changes#versions for more details
```

**Comment by rikroe** [comment]
Thanks, is fixed with [20230604.1](https://github.com/bimmerconnected/ha_custom_component/releases/tag/20230604.1).

**Comment by juan11perez** [comment]
Thank you
